.github/workflows/test.yml

```
@@ -37,9 +37,17 @@ jobs:
         include:
           - container: ghcr.io/oca/oca-ci/py3.10-odoo17.0:latest
             name: test with Odoo
+            exclude: "mail_send_confirmation"
           - container: ghcr.io/oca/oca-ci/py3.10-ocb17.0:latest
             name: test with OCB
             makepot: "true"
+            exclude: "mail_send_confirmation"
+          - container: ghcr.io/oca/oca-ci/py3.10-odoo17.0:latest
+            name: test with Odoo
+            include: "mail_send_confirmation"
+          - container: ghcr.io/oca/oca-ci/py3.10-ocb17.0:latest
+            name: test with OCB
+            include: "mail_send_confirmation"
```

```suggestion
            include: "mail_send_confirmation"
    env:
      INCLUDE: "${{ matrix.include }}"
      EXCLUDE: "${{ matrix.exclude }}"
```

oops, forgot that part